pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -2314,65 +2314,36 @@ status:
 	}
 }
 
-func TestReconcilePodFailuresStepImagePullFailed(t *testing.T) {
```

Comment:
I turned the next test into a matrix test, and the two cases in here are included in the next test now.